library/lua/3rdparty/alt_getopt.lua

```diff
@@ -0,0 +1,169 @@
+-- Copyright (c) 2009 Aleksey Cheusov <vle@gmx.net>
```

Comment:
I was about to suggest `internal/alt_getopt.lua` like the JSON library, but it looks like I named the original implementation `json/internal.lua` instead of `internal/json.lua` - guess we don't have a standard for this after all. `3rdparty` is good with me (maybe more accurate too), as long as starting module names with numbers doesn't cause issues (Python would complain, but I think Lua doesn't care).

The other third-party module I know of is `dumper.lua`, but that's different because it isn't wrapped at all, so putting it in the top-level `library/lua` folder makes sense.